### PR TITLE
feat(cli): Add docs error link to errors in the CLI

### PR DESCRIPTION
**Pull request by Princesseuh**
## Changes

Adds a link to the docs's error reference in the CLI, much like in the overlay

image

## Testing

Tested manually

## Docs

N/A

### CODE REVIEW

File: packages/astro/src/core/errors/dev/utils.ts [hunk 1]
```diff
@@ -206,13 +207,29 @@ function cleanErrorStack(stack: string) {
...
+}
+
 /**
  * Render a subset of Markdown to HTML or a CLI output
  */
 export function renderErrorMarkdown(markdown: string, target: 'html' | 'cli') {
 	const linkRegex = /\[(.+)\]\((.+)\)/gm;
 	const boldRegex = /\*\*(.+)\*\*/gm;
-	const urlRegex = / (\b(https?|ftp):\/\/[-A-Z0-9+&@#\\/%?=~_|!:,.;]*[-A-Z0-9+&@#\\/%=~_|]) /gim;
+	const urlRegex = /(\b(https?|ftp):\/\/[-A-Z0-9+&@#\\/%?=~_|!:,.;]*[-A-Z0-9+&@#\\/%=~_|])/gim;
```
Comment (Princesseuh) [inline packages/astro/src/core/errors/dev/utils.ts:232]: Unrelated fix, the regex required a space on both ends of the link, which broke the underline in some cases, I'm not quite sure why I did it that way in the first place..

**Review by ematipico — APPROVED** [review]
It worked! :) Sorry for the unsound regression I caused, thank you for fixing it
